src/components/SecondaryPanes/Expressions.css

```diff
@@ -11,8 +11,11 @@
   margin: 0;
   border: 1px;
   background-color: var(--theme-sidebar-background);
-  font-size: 12px;
```

Comment:
With this rule gone, the input text uses the system font-size in Firefox on Linux. That's 16px in my case, a bit big in the context of DevTools. :)
A `font-size: inherit` or even `font: inherit` might help.

Reply:
Changed this to `font-size: inherit` instead.